rasa_core/training_utils/dsl.py

```diff
@@ -128,6 +128,8 @@ def as_story_string(self, flat=False):
         for s in self.events:
             if isinstance(s, UserUttered):
                 result += "* {}\n".format(s.as_story_string())
+            elif isinstance(s, BotUttered):
```

Comment:
Ok either
- change this so that the ignore happens in the `elif isinstance(s, Event)` case
```
elif isinstance(s, Event):
   converted = s.as_story_string()
   if converted:
     result += "    - {}\n".format(converted)
```
- or let `BotUttered.as_story_string()` throw an exception again

Otherwise, there is a `as_story_string` that never gets called.